.github/workflows/build.yml

```
@@ -17,22 +17,17 @@ jobs:
         runs-on: ubuntu-latest
 
         name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, 
-  MySQL ${{ matrix.mysql }}, Node ${{ matrix.node }}"
+  MySQL ${{ matrix.mysql }}, State Machine Adapter ${{ matrix.state_machine_adapter }}"
 
         strategy:
             fail-fast: false
             matrix:
-                php: [ "8.1", "8.2", "8.3" ]
-                symfony: ["^5.4", "^6.4"]
-                sylius: [ "~1.13.0", "~1.14.0" ]
-                node: [ "18.x", "20.x" ]
-                mysql: ["8.0"]
-
-                exclude:
-                    -
-                        php: "8.3"
-                        symfony: "^5.4"
-                        sylius: "~1.13.0"
+                php: [ "8.2", "8.3" ]
+                symfony: [ "^7.1" ]
+                sylius: [ "^2.0" ]
+                node: [ "20.x" ]
+                mysql: [ "8.0" ]
+                state_machine_adapter: ["winzou_state_machine", "symfony_workflow"]
```

```suggestion
                state_machine_adapter: ["symfony_workflow"]
```
and include single build with winzou